.github/workflows/ci.yml

```diff
@@ -355,3 +356,36 @@ jobs:
       - name: Verify that Tokio version is up to date in README
         working-directory: tokio
         run: grep -q "$(sed '/^version = /!d' Cargo.toml | head -n1)" README.md
+
+  test-hyper:
+    name: Test hyper
+    runs-on: ubuntu-latest
```

Comment:
Could we run hyper tests on other platforms as well?

Reply:
#4393 (CI part of this PR) contains changes for that.